roles/kubernetes/control-plane/defaults/main/main.yml

```
@@ -236,3 +236,6 @@ kube_apiserver_tracing_sampling_rate_per_million: 100
 
 # Enable kubeadm file discovery if anonymous access has been removed
 kubeadm_use_file_discovery: "{{ remove_anonymous_access }}"
+
+# Set the asymmetric encryption algorithm for the cluster's keys and certificates
+kube_asym_encryption_algorithm: "RSA-2048"
```

```suggestion
kube_asymmetric_encryption_algorithm: "RSA-2048"
```
Not worth saving some character over this.
Also, can you add a link here to the relevant Kubernetes documentation ?

---

done